api/src/test/java/io/kaoto/backend/api/resource/v1/IntegrationsResourceIT.java

```diff
@@ -0,0 +1,101 @@
+package io.kaoto.backend.api.resource.v1;
+
+import static io.restassured.RestAssured.given;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+
+import javax.ws.rs.core.Response;
+
+import org.junit.jupiter.api.Test;
+import org.yaml.snakeyaml.Yaml;
+
+import io.quarkus.test.common.http.TestHTTPEndpoint;
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+
+@TestHTTPEndpoint(IntegrationsResource.class)
+@QuarkusIntegrationTest
+class IntegrationsResourceIT {
+
+	@Test
+	void expressionObject() throws Exception {
```

Comment:
Could this be like `new IntegrationsResourceTest().expressionObject()`? we'd see refactorings around expression sooner and this test would be revised repeatedly, and keep them sync between JVM<>native test would help

Reply:
it could be. the ideal would be to find a way to just inherit from the IntegrationsResourceTest and deal with the problem of the @Inject. Maybe need to separate few methods. Maybe just overriding the @before/@after methods. I provided an issue to tackle it some time ago https://github.com/KaotoIO/kaoto-backend/issues/437

Reply:
Now it's perfect :+1: